app/src/main/java/com/example/contentprovider_example/ui/adapter/EmployeeAdapter.kt

```diff
@@ -0,0 +1,31 @@
+package com.example.contentprovider_example.ui.adapter
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.example.contentprovider_example.R
+import com.example.contentprovider_example.data.model.Employee
+import kotlinx.android.synthetic.main.employee_item.view.*
+
+class EmployeeAdapter(private val employeeList: ArrayList<Employee>)
+    : RecyclerView.Adapter<EmployeeAdapter.EmployeeViewHolder>() {
```

Comment:
lần sau viết như này nhé 
```suggestion
class EmployeeAdapter(
    private val employeeList: ArrayList<Employee>
) : RecyclerView.Adapter<EmployeeAdapter.EmployeeViewHolder>() {
```

Reply:
yepp